### ISSUE TITLE
[Discussion] Opinions for a new video container format to be used in Lenstube

### ISSUE THREAD

**Issue by danimesq** [tapexyz/tape#409]
As I've proposed to [DTube](https://github.com/dtube/dtube/issues/142):

> Back in 2017, seeing the interactivity limitation in YouTube videos, and also lack of ergonomic accessibility, I thought in a brand new video format: USV. In the same epoch, it gained a site and its first specification docs. It should have multiple subtitles and even audio in different languages, and interactive options (since 2010 I'm proponent of a interactive TV) such as scene selection, polls and like/dislike buttons appearing on-screen programatically.
> 
> 2019 gave a total change in USV's direction, and I saw the previous concept was very wrong: built-in files (which would make it very similar to the heavy MKV). Instead, it should use the IPFS CID of the video scenes and audios, downloading asynchronously, and also containering existing video formats instead of frames. The USV Player, a existing app since the start of USV, gained a update capable of reading and displaying basic video data: title, cover, family rating and languages.
> 
> I'm yet open on improving USV Player and making it compatible to be used in DTube. The specification isn't fully developed, so opinions are welcome.

https://usv.js.org/
